aosp_diff/preliminary/system/sepolicy/11_0011-Allow-adbd-to-access-proc-net-tcp-tcp6-udp-udp6-.patch

```diff
@@ -0,0 +1,50 @@
+From b2fd7a5acc84b1b21563ccc893a97bb93b33d64c Mon Sep 17 00:00:00 2001
+From: Gregory Montoir <gmontoir@google.com>
+Date: Mon, 6 Sep 2021 09:25:13 +0800
+Subject: [PATCH] Allow adbd to access /proc/net/{tcp,tcp6,udp,udp6}
+
+File accesses go through com.android.ddmlib.SyncService for CTS
+ListeningPortsTest.
```

Comment:
This is also already applied.
Project ./system/sepolicy
 Already applied         11_0011-Allow-adbd-to-access-proc-net-tcp-tcp6-udp-udp6-.patch